config/locales/es.yml

```diff
@@ -4042,6 +4042,12 @@ es:
           taxes_owed_html: Tu reembolso es $0. Esto significa <strong>que no</strong> recibirás un reembolso estatal y <strong>no</strong> debes pagar impuestos estatales. Debes completar tu declaración, es posible que tengas que presentarla aunque no recibirás ningún dinero.
           title: "¡Ya casi terminas! Por favor revisa la información que tenemos."
           your_name: Tu nombre
+        az_retirement_income_deductions_review_header:
+          government_pension: Qualifying federal, Arizona, or local government pension plan
+          none_apply: None apply
+          subtitle: These allow people to deduct all or a portion of their retirement income from their taxes.
+          title: Arizona retirement income deductions
+          uniformed_services: Retired or retainer pay of the uniformed services of the United States
```

Comment:
can we check with product about the spanish translations? my guess is that they may have them & just haven't linked it to the story.

Reply:
left a q https://codeforamerica.atlassian.net/browse/FYST-902?focusedCommentId=16444